cypress/integration/list.js

```diff
@@ -15,6 +15,22 @@ describe('List Page', () => {
         });
     });
 
+    describe('Headroom', () => {
+        it('should show the correct title in the appBar', () => {
+            cy.get(ListPagePosts.elements.title).contains('Posts List');
+        });
+
+        it('should hide the appBar when scroll down action appears', () => {
+            cy.scrollTo(0, 500);
+            cy.get(ListPagePosts.elements.headroom).should('not.be.visible');
+        });
+
+        it('should show the appBar when scroll top action appears', () => {
+            cy.scrollTo(0, -500);
```

Comment:
Wouldn't it be better to only scroll up a little (less than 500, maybe only 200) ?

Reply:
My understanding is that it should appear when you start scrolling up, right ?

Reply:
- Hep, I can 200px or less (like 100px), I try in local

- Yes exactly